static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
Yeah also good to change `options.contacts`. This all dates back to the original screens where Messages were called Contacts (before the new Contacts existed).